polaris-migrator/src/cli.ts

```diff
@@ -0,0 +1,112 @@
+import fs from 'fs';
+import path from 'path';
+
+import chalk from 'chalk';
+import isGitClean from 'is-git-clean';
+import globby from 'globby';
+import meow from 'meow';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Using `@ts-expect-error` instead of ts-ignore should keep eslint happy

```suggestion
// @ts-expect-error Some Description of why this throws an error here
```

Reply:
Ah nice! Great suggestion, fixed 👍